lock-api/src/main/java/com/palantir/lock/client/metrics/TimeLockFeedbackBackgroundTask.java

```diff
@@ -69,16 +77,20 @@ public static TimeLockFeedbackBackgroundTask create(TaggedMetricRegistry taggedM
     public void scheduleWithFixedDelay() {
         executor.scheduleWithFixedDelay(() -> {
             try {
+                ConjureTimeLockClientFeedback feedbackReport = ConjureTimeLockClientFeedback
+                        .builder()
+                        .stats(ImmutableMap.of("conjureTimelockServiceBlocking.startTransactions",
+                                getEndpointStatsForStartTxn()))
+                        .atlasVersion(versionSupplier.get())
+                        .nodeId(nodeId)
+                        .serviceName(serviceName)
+                        .build();
+                timeLockClientFeedbackServices
+                        .get()
+                        .forEach(service -> service.reportFeedback(AUTH_HEADER, feedbackReport));
```

Comment:
I think we should be resilient to the first node in the cluster being unavailable.